### ISSUE TITLE
Pytorch 實作 CBOW with Hierarchical Softmax - seed9D's blog

### ISSUE THREAD

**Issue by seed9D** [seed9D/seed9D.github.io#10]
https://seed9d.github.io/Pytorch-Implement-CBOW-with-Hierarchical-Softmax/ 

 CBOW with Hierarchical SoftmaxCBOW  的思想是用兩側 context words 去預測中間的 center word P(center|context;\theta)
